ledger_hid.go

```diff
@@ -42,6 +41,12 @@ type LedgerDeviceHID struct {
 	readChannel chan []byte
 }
 
+// list of supported product ids as well as their corresponding interfaces
+var supportedLedgerProductID = map[uint16]int{
+	0x4011: 0, // Ledger Nano X
```

Comment:
The problem with this is that in the past, these values have changed according to the firmware version.
While they are supposed to be stable in the future, we should keep an eye on them in case there are any breaking changes.

Reply:
Should be pretty easy to add new/updated values.